deploy/templates/notificationtemplates/sandbox/userprovisioned/notification.html

```diff
@@ -41,27 +41,26 @@
 >
 
     <p>
-        You are receiving this email because you have a Developer Sandbox for Red Hat OpenShift
-        account associated with {{.UserEmail}}.
+        You are receiving this email because the Developer Sandbox account associated with {{.UserEmail}} 
+        is now provisioned and ready to use. Your account will be active for {{.DeactivationTimeoutDays}} days at no cost to you.
```

Comment:
any reason for splitting the sentence into two lines when all other sentences are single-line only?
```suggestion
        You are receiving this email because the Developer Sandbox account associated with {{.UserEmail}} is now provisioned and ready to use.
        Your account will be active for {{.DeactivationTimeoutDays}} days at no cost to you.
```

Reply:
It won't split in the actual email - please see screenshot.